### DISCUSSION TITLE
Update to DICOM standars 2021c

### DISCUSSION THREAD

**Discussion by maropapa** [fo-dicom/fo-dicom#1257] [General]
Hi there,

Do you plan to upgrade to DICOM 2021c, especially Microscopy Bulk Simple Annotations IOD? What I really miss is the SOP Class UID. If it will not happen soon please send some advice how to add the UID manually.

Thanks,
maropapa

**Comment by gofal** [comment]
sure, I will update to current DICOM standard with next release.
The UIDs are generated via a T4 script, that downloads the definitions from nema-server, extracts all uids and then generates the cs-files. See here, how it works:  https://github.com/fo-dicom/fo-dicom/wiki/DICOM-dictionary-generation
so you can get the current source and do this update yourself.
